core/imageroot/usr/local/agent/bin/module-backup

```diff
@@ -27,6 +27,25 @@ import os
 import os.path
 import subprocess
 import time
+import atexit
+
+backup_status = {}
+backup_status['total_size'] = 0
+backup_status['total_file_count'] = 0
+backup_status['snapshots_count'] = 0
```

Comment:
Since we are doing a little refactor, let's cleanup the code a little bit.

```suggestion
backup_status = {'total_size': 0, 'total_file_count': 0, 'snapshots_count': 0, 'errors': 0, 'start': int(time.time())}
```